src/pcre2_compile.c

```diff
@@ -10362,6 +10365,9 @@ int regexrc;                          /* Return from compile */
 
 uint32_t i;                           /* Local loop counter */
 
+/* Enable all optimizations by default. */
+uint32_t optim_flags = ccontext != NULL ? ccontext->optimization_flags : 0x7;
```

Comment:
0x7 should be a `#define` so it can be updated more easily

Reply:
You are absolutely right. This even crossed my mind after I pushed this PR.

Reply:
Applied and Carlo credited as co-author.
Regarding the idea of just inverting the mask to get "full optimization", I actually thought of even just using `0xffffffff` or something like that. But I decided not to. I think this value can be updated as more optimization flag bits are defined.